### PR TITLE
Update dictionary.txt

### PR DESCRIPTION
**Pull request by arm-in**
Some new entries collected from production code.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -943,6 +943,7 @@ automaitc->automatic
 automaitcally->automatically
 automaticall->automatically
 automaticaly->automatically
+automatical->automatically, automatic, automated
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: missing a comma at the end of `automated`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -6654,7 +6662,8 @@ propogates->propagates
 propogating->propagating
 propogation->propagation
 propostion->proposition
-propotions->proportions
+propotion->proportion, promotion
+propotions->proportions, promotions
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: missing commas at the end of each of these

**Comment by arm-in** [comment]
Thanks for reviewing. My respect, you have a sharp sight! What do I have to do now? Should I rework the pull request?

**Comment by luzpaz** [comment]
@arm-in revise it by making the appropriate changes and then pushing those changes (no need to make a new PR)

**Comment by luzpaz** [comment]
LGTM

**Comment by larsoner** [comment]
Thanks @arm-in

**Pull request merged**